paddle/fluid/imperative/engine.cc

```diff
@@ -195,16 +196,21 @@ void BasicEngine::Execute() {
     NameVarBaseMap tmp_outs;
     // A var may be coresponding to several grad var in one op
     std::unordered_map<VarBase*, std::vector<std::shared_ptr<VarBase>>> var_map;
-    size_t counter = 0;
+    // size_t counter = 0;
```

Comment:
If the code is not necessary, please remove them directly instead of commenting.

Reply:
done